Use proper server name in repoman configuration

oliverguenther opened the pull request:
Repoman is not accessible from localhost in all configurations,
thus we instead use the proper servername configured in the wizard
to communicate with repoman for SVN repositories.

Adds an `insecure` option to the `configuration.yml`.

This option only applies when the manages: key is set to a URL.

If the given URL uses SSL, certificate checking will be disabled.
This key is set for a packaged installation by default, since it communicates locally and
these installations may often include a snakeoil certificate.

Related to https://github.com/finnlabs/addon-repositories/pull/5

/cc @crohr

oliverguenther committed:
Use proper server name in repoman configuration

Repoman is not accessible from localhost in all configurations,
thus we instead use the proper servername configured in the wizard
to communicate with repoman for SVN repositories.

Related to https://github.com/finnlabs/addon-repositories/pull/5

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://oliverguenther-fix-repoman-localhost.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://oliverguenther-fix-repoman-localhost.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

oliverguenther committed:
Add insecure option to managed remotes

Adds an `insecure` option to the `configuration.yml`.

This option only applies when the manages: key is set to a URL.

If the given URL uses SSL, certificate checking will be disabled.
This key is set for a packaged installation by default, since it communicates locally and
these installations may often include a snakeoil certificate.
